### ISSUE TITLE
Transfer Learning Lab: Cifar10 Aside need more information

### ISSUE THREAD

**Issue by HenryDev** [udacity/sdc-issue-reports#473]
It says "the only thing you'll need to change is the number of classes from 43 to 10." but it's not plug and play and there's more modification needed for this to work. The project comes with validation set which cifar10 doesn't, so the parameters need to be changed when evaluating. Also the placeholders in feed_dict may need adjustments as well so they're in the same shape. I'm getting `Cannot feed value of shape (128, 3, 32, 32) for Tensor 'Placeholder_12:0', which has shape '(?, 32, 32, 3)'` when I try to do the training operation.

**Comment by andrewpaster** [comment]
Hey thanks for pointing this out. I realize you've probably moved on from this part of the classroom by now. I'll mention the validation set in that part of the lecture. 

Not sure about the Tensorflow issue. When I load the cifar10 data set, I get an array [50000, 32, 32, 3], which would match your tensorflow placeholder. 

Weird that it would've read in the cifar10 data set as a 3 x 32 x 32. When using Theano as the backend, which is another deep learning library, Keras will use 3 x 32 x 32 instead of 32 x 32 x 3.
